src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""Factory functions for all ngraph ops."""
```

Comment:
```suggestion
"""Factory functions for all openvino ops."""
```
Shouldn't we switch from ngraph to ov?

Reply:
Aligned with the second suggestion